modules/auxiliary/server/relay/smb_to_ldap.rb

```diff
@@ -0,0 +1,118 @@
+##
```

Comment:
Maybe adding a check for the ldap session feature until it is enabled by default.

Reply:
Yes, very good idea. I added a check in the last commit. Thank you!